### DISCUSSION TITLE
Functions: support dynamic express route parameters

### DISCUSSION THREAD

**Discussion by njbmartin** [gatsbyjs/gatsby#34179] [Ideas / Feature Requests]
Looking at this PR, named splat routes were added:
https://github.com/gatsbyjs/gatsby/pull/33301

eg. `/api/customers/[...customerId].ts`

This is great, but it doesn't look like dynamic routes in Functions are fully supported yet.

Moving from a pure express API to Gatsby Functions, I can't seem to do `/api/auth/[customerId]/[customerEmail].ts` as these return undefined.

**Comment by njbmartin** [comment]
Actually, this might be because it's not yet supported in Gatsby Cloud?

**Comment by KyleAMathews** [comment]
Hmm does it work locally?

**Reply by njbmartin** [reply]
Yes, it works locally, just not in Cloud so had to revert the build
